README.md

```diff
@@ -4,12 +4,12 @@
 [![Known Vulnerabilities](https://snyk.io/test/github/laniksj/slack-dark-mode/badge.svg?targetFile=/docs/Gemfile.lock)](https://snyk.io/test/github/laniksj/slack-dark-mode?targetFile=/docs/Gemfile.lock)
 
 ## Purpose
-Store scripts and Style sheets for Slack Dark Mode for macOS Mojave.  
-Dark Mode in Slack isn't available as of this writing.  
+Store scripts and Style sheets for Slack Dark Mode for macOS Mojave.
+Dark Mode in Slack isn't available as of this writing.
 
 ## Notice
-Due to the changes in Slack 4.0+ this project will not be compatible with Slack Version 3.4 or below.   
-If you're looking for 3.4.x compatible settings please refer to [this](https://github.com/LanikSJ/slack-dark-mode/tree/466ff22d5b606b6d5b2edeff54f4cd7a3bafc39c).   
+Due to the changes in Slack 4.0+ this project will not be compatible with Slack Version 3.4 or below.
+If you're looking for 3.4.x compatible settings please refer to [this](https://github.com/LanikSJ/slack-dark-mode/tree/466ff22d5b606b6d5b2edeff54f4cd7a3bafc39c).
```

Comment:
Same as the above comment.

Reply:
should be fixed

Reply:
also fixed in a 3rd location that you missed at the bottom.